libraries/AP_BattMonitor/AP_BattMonitor_SMBus_SUI.cpp

```diff
@@ -22,12 +22,14 @@ AP_BattMonitor_SMBus_SUI::AP_BattMonitor_SMBus_SUI(AP_BattMonitor &mon,
       cell_count(_cell_count)
 {
     _pec_supported = false;
-    _dev->set_retries(2);
 }
 
 void AP_BattMonitor_SMBus_SUI::init(void)
 {
     AP_BattMonitor_SMBus::init();
+    if (_dev) {
+        _dev->set_retries(2);
```

Comment:
Arguably should be inside the next if, as if there isn't a timer handle it's not like we will be reading any data anyways.

Reply:
yeah, not sure why it wasn't just combined